babblesim/nrfx/mdk/nrf52.h

```diff
@@ -176,6 +176,7 @@ typedef enum {
   #warning Not supported compiler type
 #endif
 
+#undef B0
```

Comment:
why?

Reply:
B0 from #include <termios.h> in babblesim/hw/mcu/nordic/nrf52_bsim/src/native_uart_cfg.c conflicts with B0 from nrfx/mdk/nrf52.h. It was possible to move this line to the native_uart_cfg.c to leave nrfx as clean as possible.